src/Sylius/Bundle/ShopBundle/EventListener/OrderPromotionIntegrityChecker.php

```diff
@@ -57,7 +56,7 @@ public function __construct(
         PromotionEligibilityCheckerInterface $promotionEligibilityChecker,
         EventDispatcherInterface $eventDispatcher,
         RouterInterface $router,
-        PromotionApplicatorInterface $promotionApplicator = null
+        ?PromotionApplicatorInterface $promotionApplicator
```

Comment:
One more thing, that default `null` value should stay as is (so it's not breaking backwards compatibility)

Reply:
Done, sorry for missunderstanding you